### ISSUE TITLE
when a new user registers, its information is shown to all visitors ?!

### ISSUE THREAD

**Issue by Coderx7** [q2a/question2answer#436]
I just noticed something strange, 
this is the picture of a newly registered user.! 
I just tried to open the site and bang faced it! : 
what is wrong here?!!
if I click on one of the tabs, it shows the ordinary unlogged version that any new visitor should see when they open the website!

**Comment by Coderx7** [comment]
ok seems like a caching

**Comment by svivian** [comment]
CloudFlare should respect the headers set by the web server. Q2A sends headers to prevent caching so not sure why that would happen even with the "aggressive caching" option.

**Comment by Coderx7** [comment]
Actually

**Comment by svivian** [comment]
According to [this](https://support.cloudflare.com/hc/en-us/articles/200169266-Does-CloudFlare-honor-my-Expires-and-Cache-Control-headers-for-static-content-) CF honours the `Expires` header, which is being sent on your site, and set to prevent caching. Might want to read through [this page](https://support.cloudflare.com/hc/en-us/articles/202775670-How-Do-I-Tell-CloudFlare-What-to-Cache-) as well to make sure you have everything set properly.

**Comment by Coderx7** [comment]
Thanks to God the problem seems to be ceased now!
The cloudFlare is completely disabled.
I also changed  the .htacess contents which was updated by a wordpress addon named wordfense. 
and I guess this one was the culprit behind all of these issues, but still I'm not sure. 
I'm writing these here, so that maybe someone in the future stumble upons this issue him/her self and can get an idea what might be the problem.
